codice/ddf
# DDF-4330 Relativize UI Imports

#### What does this PR do?

Relativize UI Imports

#### Who is reviewing it?

@adimka 
@gjvera 

#### Select relevant component teams: 

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@rzwiefel

#### How should this be tested?

- `mvn clean install` ui directory
- ensure no ui regressions
- ensure no ui regression in downstream

#### What are the relevant tickets?

[DDF-4330](https://codice.atlassian.net/browse/DDF-4330)
